joomla-projects/privacy-framework
# Allow logout without accept privacy policy

Pull Request for Issue #122 .

### Summary of Changes
This PR fixes the issue #122, allow users to logout without accepting privacy policy


### Testing Instructions
See https://github.com/joomla-projects/privacy-framework/issues/122 for issue description. Apply patch, confirm the issue fixed.
